server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/single_page_apps/pipeline_configs/_dashboard_view_tabs.scss

```diff
@@ -149,30 +150,21 @@ $dashboard-tabs-box-shadow: 0 0 10px rgba(0, 0, 0, 0.21);
 }
 
 .dashboard-tabs-sortable, .dashboard-tabs-scrollable {
-  position:       relative;
-  margin:         0;
-  padding:        0;
   vertical-align: top;
   white-space:    nowrap;
 }
 
 .dashboard-tabs-sortable {
-  padding-bottom: 100px;
-  width:          100%;
-  overflow-x:     auto;
-  overflow-y:     hidden;
-  outline:        none;
+  overflow: hidden;
```

Comment:
Removing the padding-bottom and setting overflow-x to hidden disables the native horizontal scroll for hardware that is capable (e.g. Mac laptops and other touch devices that allow horizontal scroll, which forces people to use the nav buttons).

Could we restore the `overflow-x: auto` plus some arbitrary `padding-bottom` (e.g. `100px`) to ensure the horizontal scrollbars are rendered outside of the clipped region?